kedro_airflow_k8s/cli.py

```diff
@@ -56,6 +57,16 @@ def _create_template_stream(
     )
 
 
+def get_dag_filename_and_template_stream(ctx, cron_expression=None):
+    context_helper = ctx.obj["context_helper"]
+    package_name = context_helper.context.package_name
+    dag_filename = f"{package_name}.py"
```

Comment:
There should be an option to override `dag_filename`, we allow running the same pipelines from different branches as an experiment, we need to avoid them overwriting themselves.

Reply:
I create additional CLI parameter that allows to override dag id. This gives both distinction in airflow console and prevents DAG definition override. Is this what you have in mind?

Reply:
Yes, exactly that.